src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -396,9 +396,8 @@ private boolean areClusteredSNPs(final FeatureContext featureContext, final Vari
     }
 
     @Override
-    public Object onTraversalSuccess() {
+    public void closeTool() {
         writer.close();
```

Comment:
good catch - now that closeTool is called on exceptional paths, this can be called the field is initialized